pkg/webhooks/kafkatopic_validator.go

```diff
@@ -55,22 +63,22 @@ func (s KafkaTopicValidator) ValidateDelete(ctx context.Context, obj runtime.Obj
 
 func (s *KafkaTopicValidator) validate(ctx context.Context, obj runtime.Object) error {
 	kafkaTopic := obj.(*banzaicloudv1alpha1.KafkaTopic)
-	log := s.Log.WithValues("name", kafkaTopic.GetName(), "namespace", kafkaTopic.GetNamespace())
-	fieldErrs, err := s.validateKafkaTopic(ctx, kafkaTopic, log)
+	s.Log = s.Log.WithValues("name", kafkaTopic.GetName(), "namespace", kafkaTopic.GetNamespace())
```

Comment:
**Question**: Are we sure we want to overwrite the stored logger here? For different topics this would be different and in that case the local instance would be better IMO.

Reply:
I think it is OK in this way because the kafkatopic controller runs sequential and not parallel

Reply:
I can let this fly, but the stored validator logger will still point to some topic after the validation already run which is IMO misleading, because it handles multiple topics.
(Practically the result is the same, but conceptually we are abusing implementation details instead of doing this cleanly.)

Reply:
Agree. Thank you for bringing this up. I fix this.